### PR TITLE
TFE - update docker on RHEL info

### PR DESCRIPTION
**Pull request by amy-hashi**
### What
Updated the RHEL specific page for TFE to better guide customers regarding Docker options.

### Why
As customers have adopted RHEL8, they are more likely to have questions about docker.

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: content/enterprise/requirements/os-specific/rhel-requirements.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ Red Hat Enterprise Linux v8 does not offer docker in the EPEL repository, please
 
 ### Docker was accidentally upgraded, how can I downgrade?
 
-The `yum downgrade` command can be used to downgrade the version of Docker that is installed.
+The `yum downgrade` command can be used to downgrade the version of Docker that is installed, if it was installed from the EPEL repository.
```
Comment (laurapacilio) [inline content/enterprise/requirements/os-specific/rhel-requirements.mdx:33]:
```suggestion
You can use the `yum downgrade` command to downgrade the version of Docker that is installed, if it was installed from the EPEL repository.
```

Comment (laurapacilio) [reply content/enterprise/requirements/os-specific/rhel-requirements.mdx:33]: If it wasn't installed this way, are they out of luck?

**Review by laurapacilio — APPROVED** [review]
Approving so I don't block but a few comments!

**Review by sudomateo — COMMENTED** [review]
Thank you for doing this Amy!

**Pull request merged**